Test/ContentPipeline/FbxImporterTests.cs

```diff
@@ -150,21 +153,21 @@ public void Dude()
             Assert.AreEqual(1.19371974f, materials["character_anim:upBodyM"].SpecularPower, 0.00001f);
             Assert.AreEqual(65.986f, materials["character_anim:eyeBallM"].SpecularPower, 0.00001f);
 
-            Paths.AreEqual(@"Assets\Models\Dude\head.tga", materials["character_anim:headM"].Textures["Texture"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\headS.tga", materials["character_anim:headM"].Textures["Specular"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\headN.tga", materials["character_anim:headM"].Textures["Bump"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\jacket.tga", materials["character_anim:jacketM"].Textures["Texture"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\jacketS.tga", materials["character_anim:jacketM"].Textures["Specular"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\jacketN.tga", materials["character_anim:jacketM"].Textures["Bump"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\pants.tga", materials["character_anim:pantsM"].Textures["Texture"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\pantsS.tga", materials["character_anim:pantsM"].Textures["Specular"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\pantsN.tga", materials["character_anim:pantsM"].Textures["Bump"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\upBodyC.tga", materials["character_anim:upBodyM"].Textures["Texture"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\upBodyS.tga", materials["character_anim:upBodyM"].Textures["Specular"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\upbodyN.tga", materials["character_anim:upBodyM"].Textures["Bump"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\upBodyC.tga", materials["character_anim:eyeBallM"].Textures["Texture"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\upBodyS.tga", materials["character_anim:eyeBallM"].Textures["Specular"].Filename);
-            Paths.AreEqual(@"Assets\Models\Dude\upbodyN.tga", materials["character_anim:eyeBallM"].Textures["Bump"].Filename);
+            Paths.AreEqual(@"Assets/Models/Dude/head.tga", materials["character_anim:headM"].Textures["Texture"].Filename);
+            Paths.AreEqual(@"Assets/Models/Dude/headS.tga", materials["character_anim:headM"].Textures["Specular"].Filename);
```

Comment:
how does this work on Windows? Does it produce `/` in our imported there too?

Reply:
If you look at my Paths.AreEqual you'll see it normalizes the slashes via Path.GetFullPath.  However it doesn't work on Mac without me flipping these.

So it works.